### PR TITLE
fix: style Recent Project improvements & clicn on whole item

### PR DESCRIPTION
**Pull request by erdkse**
This PR...

## Changes

-

## Fixes

- Recent project; instead of clicking title, click on the whole item

## How to test it

-

## Screenshots

-

## Checklist

- [ ] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

### CODE REVIEW

File: src/components/organisms/RecentProjectsPane/RecentProjectsPane.tsx [hunk 1]
```diff
@@ -47,13 +47,13 @@ const RecentProjectsPane = () => {
...
-                <S.ProjectName
-                  onClick={() => (isActivePropject ? dispatch(toggleStartProjectPane()) : openProject(project))}
-                >
-                  {project.name}
-                </S.ProjectName>
-                <Tooltip title={project.rootFolder}>
+              <S.ProjectItem
+                key={project.rootFolder}
+                activeproject={isActivePropject}
+                onClick={() => (isActivePropject ? dispatch(toggleStartProjectPane()) : openProject(project))}
```
Comment (DmitryEremenko) [inline src/components/organisms/RecentProjectsPane/RecentProjectsPane.tsx:53]: Please don't do any logic in onClick handler. Create separate handler for it and use it inside

**Review by DmitryEremenko — CHANGES_REQUESTED** [review]

File: src/components/organisms/PaneManager/PaneManager.tsx [hunk 2]
```diff
@@ -215,7 +215,7 @@ const PaneManager = () => {
             <StartProjectPane />
           </div>
           {Boolean(projects.length) && (
-            <div style={{flex: 1, borderLeft: `1px solid ${Colors.grey3}`}}>
+            <div style={{flex: 1, borderLeft: `1px solid ${Colors.grey3}`, maxWidth: '28vw'}}>
```
Comment (topliceanurazvan) [inline src/components/organisms/PaneManager/PaneManager.tsx:218]: Can we create a styled container ( div ) inside `RecentProjectsPane` and remove the inline styling from the div here?

Comment (erdkse) [reply src/components/organisms/PaneManager/PaneManager.tsx:218]: yeah of course

Comment (topliceanurazvan) [reply src/components/organisms/PaneManager/PaneManager.tsx:218]: Perfect 👍🏻

**Review by topliceanurazvan — APPROVED** [review]
Looks great!

**Review by devcatalin — APPROVED** [review]

**Pull request merged**